### PR TITLE
Fixes webextensions.api.windows.create notes issues

### LINKED ISSUE #24888: webextensions.api.windows.create - desc about `focused: false` is misleading
### What type of issue is this?

Incorrect support data (example: BrowserX says "86" but support was added in "40")

### What information was incorrect, unhelpful, or incomplete?

The description gives an impression that `focused: false` works normally in Firefox < 86, while it actually causes an error to be thrown in those versions.

### What browsers does this problem apply to, if applicable?

Firefox

### What did you expect to see?

Should have described that `focused: false` causes an error in Firefox < 86.

### Did you test this? If so, how?

I confirmed that `focused: false` causes an error in Firefox 68.

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN URL

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/windows/create

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `webextensions.api.windows.create`
* Report started: 2024-10-31T03:08:32.913Z

</details>

### LINKED ISSUE #24889: webextensions.api.windows.create - missing ending quote for tabId for Firefox
### What type of issue is this?

Other (typo fix)

### What information was incorrect, unhelpful, or incomplete?

The description about tabId for Firefox has a missing ending quote.

### What browsers does this problem apply to, if applicable?

_No response_

### What did you expect to see?

The tabId should be quoted correctly.

### Did you test this? If so, how?

Typo fix doesn't need a test.

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN URL

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/windows/create

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `webextensions.api.windows.create`
* Report started: 2024-10-31T03:14:52.362Z

</details>

### PR DESCRIPTION
**Pull request by rebloor**
#### Summary

Correct a typo and clarify a note for windows.create.

#### Related issues

Fixes #24888
Fixes #24889

### CODE REVIEW

**Review by dotproto — APPROVED** [review]
Looks good. Thanks @rebloor!

File: webextensions/api/windows.json [hunk 1]
```diff
@@ -667,9 +667,9 @@
               "firefox": {
                 "version_added": "45",
                 "notes": [
-                  "'url' and 'tabId options can't both be set together.",
-                  "The returned 'Window' object contains the 'tabs' property only from version 52 onwards.",
-                  "From Firefox 86, the <code>focused: false</code> option is ignored.",
+                  "<code>url</code> and <code>tabId</code> options can't be set together.",
+                  "The returned <code>Window</code> object contains the <code>tabs</code> property only from version 52 onwards.",
+                  "In Firefox 85 and earlier, the <code>focused: false</code> is unsupported. From Firefox 86, <code>focused: false</code> is ignored",
```
Comment (caugner) [inline webextensions/api/windows.json:672]: typo. What is the difference between unsupported and ignored?

```suggestion
                  "In Firefox 85 and earlier, <code>focused: false</code> is unsupported. From Firefox 86, <code>focused: false</code> is ignored.",
```

**Review by caugner — DISMISSED** [review]

File: webextensions/api/windows.json (hunk 1 above)
Comment (rebloor) [reply webextensions/api/windows.json:672]: Clarified - no supported means an error if you use it.
```suggestion
                  "In Firefox 85 and earlier, <code>focused: false</code> is unsupported and throws an error. From Firefox 86, <code>focused: false</code> is ignored.",```

File: webextensions/api/windows.json [hunk 2]
```diff
@@ -667,9 +667,9 @@
               "firefox": {
                 "version_added": "45",
                 "notes": [
-                  "'url' and 'tabId options can't both be set together.",
-                  "The returned 'Window' object contains the 'tabs' property only from version 52 onwards.",
-                  "From Firefox 86, the <code>focused: false</code> option is ignored.",
+                  "<code>url</code> and <code>tabId</code> options can't be set together.",
```
Comment (queengooborg) [inline webextensions/api/windows.json:670]: We just recently switched to allowing Markdown formatting in notes, so let's use that now!

**Review by queengooborg — CHANGES_REQUESTED** [review]

File: webextensions/api/windows.json (hunk 2 above)
Comment (rebloor) [reply webextensions/api/windows.json]: @queengooborg Completed while resolving merge conflicts

**Review by caugner — APPROVED** [review]

**Comment by rebloor** [comment]
@queengooborg a reminder that I've addressed your changes requested

**Pull request merged**